src/util/constants.ts

```diff
@@ -1 +1,3 @@
 export const APP_NAME = 'veilarbvedtaksstottefs';
+
+export const BEHANDLINGSNUMMER = 'B642';
```

Comment:
Kva er behandlingsnummer? Er det ein verdi som er litt hemmeleg, eller kan den liggje opent for alle på nett?